src/components/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+class Navbar extends React.PureComponent {
+  render() {
+    return (
+      <div className="nav-container">
+        <nav className="navbar">
+          <Link className="web-logo" to="/">
+            BookStore CMS
+          </Link>
+          <ul className="nav-items">
+            <li className="nav-item">
+              <Link className="nav-link active" to="/">
+                BOOKS
+              </Link>
+            </li>
+            <li className="nav-item">
+              <Link className="nav-link" to="/">
+                CATEGORIES
+              </Link>
```

Comment:
- [ ] In the Link component provided by react-router-dom, kindly pass a path of `/categories` instead of the home `/` path as this will route to the categories page, or should I say, Category page as you have in in your pages directory 👍 
- It should be like this
```jsx
<Link className="nav-link" to="/category">
     CATEGORIES
 </Link>
```